operating/view-dashboard.adoc

```diff
@@ -0,0 +1,63 @@
+[id="view-dashboard"]
+= Viewing the dashboard
+include::modules/common-attributes.adoc[]
+:context: view-dashboard
+
+toc::[]
+
+[role="_abstract"]
+{product-title} dashboard enables you to visually track and analyze key metrics such as risk across your environment, compliance status, and policy violations.
+The *Dashboard* view helps you to understand your cloud-native infrastructure, including images, containers, pods, namespaces, clusters, and their configurations.
+
+[NOTE]
+====
+When you open the RHACS portal for the first time, you may see a blank dashboard.
+After you deploy Sensor in at least one cluster, the dashboard reflects the status of your environment.
+====
+
+The *Dashboard* view includes multiple interactive widgets which show a summary of system violations, summary of compliance status, violations by cluster, top risky deployments, and active violations.
+It also includes other widgets that are automatically generated when you create new policy categories.
+//TODO: Add link to policy categories
+
+[id="system-violations"]
+== System Violations
+
+The *System Violations* widget (upper-left) shows the number of critical, high, medium, and low severity violations on the *Dashboard* view.
+You can select a number on the widget to view details about those violations.
+
+[id="compliance"]
+== Compliance
+
+The *Compliance* widget (upper-right) shows status of various compliance benchmarks and if their controls are passing or failing.
+
+[NOTE]
+====
+The *Compliance* widget show details only after you run a compliance scan.
+====
+//TODO: Add link to compliance scan
+
+[id="violations-by-cluster"]
+== Violations by Cluster
+
+The *Violations by Cluster* widget shows a bar chart based on the number of violations in each cluster in different colors based on their severity.
+
+[id="top-risky-deployments"]
+== Top Risky Deployments
+
+The *Top Risky Deployments* widget displays the top five deployments at risk in your containerized environment.
+
+You can select a deployment in this widget to get more details about the issues with that particular deployment.
+You can also select *View All* to view risks for all your deployments.
+
+From here, you can select the separate deployments to see the details of the issues found with each particular deployment.
+
+[id="active-violations-by-time"]
+== Active Violations by Time
+
+The *Active Violations by Time* widget shows the scale of number of violations by recent days, with the colors indicating the severity of those violations.
+
+[id="configurable-dashboard-widgets"]
+== Configurable Dashboard Widgets
```

Comment:
If this is not an actual widget name, should "Configurable Dashboard Widgets" be "Configurable dashboard widgets"?